.github/workflows/publish-to-pypi.yml

```diff
@@ -11,13 +11,13 @@ on:
 
 jobs:
   build-n-publish:
-    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
-    runs-on: ubuntu-18.04
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2
         with:
-          fetch-depth: 20
+          fetch-depth: 50
```

Comment:
Why did you modify "fetch-depth"?

Reply:
2.0.0rc4 development went to .dev122, so I think it may be better to increase a depth for the test with test.pypi.org.
It is not mandatory.

Reply:
Oh, I also modify to run uploading to test.pypi.org on releases branch, that should not be such development count.
OK, we can keep it as is.